Common/include/CConfig.hpp

```diff
@@ -2095,6 +2096,12 @@ class CConfig {
    */
   string GetFEA_FileName(void) const { return FEA_FileName; }
 
+  /*!
+    * \brief Determine if advanced features are used from the element-based FEA analysis (experimental feature).
+    * \return <code>TRUE</code> is experimental, <code>FALSE</code> is the default behaviour.
+    */
+  inline bool GetAdvanced_FEAElementBased(void){ return FEAAdvancedMode; }
```

Comment:
LGTM. But const please.

Reply:
Thanks @pcarruscag, nice catch! I had missed it.